nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -0,0 +1,237 @@
+"""
+Transformer for computing seeds signals.
+"""
+
+import numpy as np
+
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.externals.joblib import Memory
+
+import nibabel
+
+from .. import _utils
+from .._utils import logger, CacheMixin
+from .._utils.niimg_conversions import is_img, check_niimg
```

Comment:
Good to mention that, I was wondering about the optimal order of masker-related merges.